libopenage/datastructure/constexpr_map.h

```diff
@@ -268,15 +238,13 @@ class ConstMap<K, V, 0> {
 	 * Fetching a value from the end element aborts as it has no entry.
 	 */
 	constexpr const inline V &fetch(const K &) const {
-		return this->missing_key();
+		// A key is not in the map.
+		// This function should never be called. If it is called,
+		// you requested a nonexisting key.
+		return *static_cast<const V*>(nullptr);
```

Comment:
Well, it does give a compile time error in constexpr context if a wrong key is requested.